### DISCUSSION TITLE
HPOOL池风险极大！

### DISCUSSION THREAD

**Discussion by hz-bk201** [Chia-Network/chia-blockchain#6933] [Support]
建议不要使用这个池子。

**Comment by loppefaaret** [comment]
From a Chia Network point of view, it is not recommended to use _ANY_ pool - they are all using their own code in some way or another to pool together!
that being said, thousands of people are using hpool solution to pool together already - even if hpool have had a bit of problems.
it is up to you to if you want to use 3rd party code, and enjoy the pooling solution from hpool
just be careful with your wallet keys!
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Management

谷歌翻译
从 Chia Network 的角度来看，不建议使用任何池 - 他们都以某种方式使用自己的代码来池化！
话虽如此，成千上万的人已经在使用 hpool 解决方案来汇集在一起 - 即使 hpool 有一些问题。
是否要使用第 3 方代码并享受 hpool 的池化解决方案取决于您
小心你的钱包钥匙！
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Management

**Comment by BrandtH22** [comment]
@hz-bk201 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
